app/controllers/stripe-setup/check-org-details/post.controller.test.js

```diff
@@ -77,4 +115,32 @@ describe('Check org details - post controller', () => {
     expect(pageData.orgCity).to.equal('London')
     expect(pageData.orgPostcode).to.equal('N1 1NN')
   })
+
+  describe('radio buttons', () => {
+    it('when `no` radio button is selected, then it should redirect to org address page', () => {
+      req.account.connectorGatewayAccountStripeProgress = { organisationDetails: false }
+
+      req.body = {
+        'confirm-org-details': 'no'
+      }
+
+      controller(req, res, next)
+
+      sinon.assert.calledWith(res.redirect, 303, '/account/a-valid-external-id/your-psp/a-valid-credential-id/update-organisation-details')
+    })
+
+    it('when `yes` radio button is selected, then it should redirect to the `choose how to process payments` page', async () => {
```

Comment:
Should this be redirect to `add psp account details redirect route` ?

Reply:
Thanks - update to the `stripe > add PSP account details redirect` route